tools/uebuild/uebuild.nim

```diff
@@ -410,7 +413,9 @@ proc build(command: CommandType, engineDir, projectDir, projectName, target, mod
     cleanModules(projectDir)
 
     (os, cpu) = uePlatformToNimOSCPU(platform)
-    if platform.toLowerAscii() == "ios":
+    if cpuOverride != nil:
+      cpu = cpuOverride
```

Comment:
This should be extracted into `uePlatformToNimOSCPU` instead of being duplicated everywhere